src/resources/BaseInterfaces.ts

```diff
@@ -13,9 +13,25 @@ export interface IdAndDisplayNameModel {
 }
 
 export interface PrivilegeModel {
```

Comment:
If you use this for internal privilege too, you need the field level in here. It can be omitted when using the level: NORMAL which is the level for normal privileges used by non coveo people.